src/scss/tom-select.bootstrap4.scss

```diff
@@ -23,21 +22,19 @@ $select-color-item: #efefef !default;
 $select-color-item-border: $border-color !default;
 $select-color-item-active: $component-active-bg !default;
 $select-color-item-active-text: #fff !default;
-$select-color-item-active-border: rgba(0,0,0,0) !default;
+$select-color-item-active-border: rgb(0 0 0 / 0%) !default;
 $select-color-optgroup: $dropdown-bg !default;
 $select-color-optgroup-text: $dropdown-header-color !default;
 $select-color-optgroup-border: $dropdown-divider-bg !default;
 $select-color-dropdown: $dropdown-bg !default;
-$select-color-dropdown-border-top: mix($input-border-color, $input-bg, 0.8) !default;
+$select-color-dropdown-border-top: mix($input-border-color, $input-bg, 0.8%) !default;
```

Comment:
Fix for latest sass release warning.

Reply:
This should be 80% and not 0.8% right? https://github.com/orchidjs/tom-select/pull/529/files